### DISCUSSION TITLE
My student GitHub pro suddenly not available

### DISCUSSION THREAD

**Discussion by goodfella47** [community/community#58972] [GitHub Education] [answered]
### Hi everyone,

Hello, 

Even though I'm in the Student Developer Pack, my student GitHub pro suddenly disappeared.
![image](https://github.com/community/community/assets/15670853/4ecf6b94-11f8-448b-ac51-9cafe83c7df9)

How can I fix this issue?

**Comment by mdnurulmomen** [comment]
In order to get the GitHub Pro badge, you need to subscribe to the GitHub Pro plan.

See which options below best apply to your situation:

If you are still a student with a valid school email address, apply for the [GitHub Student Developer pack](https://education.github.com/pack) - which provides you free access to GitHub Pro (while you remain a student).
If you are no longer a student, you can subscribe to GitHub Pro on your own which should cost $4 USD monthly.
For steps on how to subscribe to GitHub Pro on your own, you can refer to: [GitHub Documentation - Step by Step instructions on how to subscribe to GitHub Pro plan](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/upgrading-your-accounts-plan#upgrading-your-personal-accounts-plan)

Additional reference: [Benefits of GitHub Pro plan](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro)

Hope this answers & resolves your question.
Take care and have a good day ahead !

**Reply by goodfella47** [reply]
Thanks for the reply, 
As I said before, I'm in the Student Developer Pack but I dont have the GitHub Pro badge.

**Comment by GhostOf0days** [comment, marked as answer]
Please check [billing](https://github.com/settings/billing/payment_information) and make sure you have a $4 off per month coupon. You should be able to go to Github Pro and [get it](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/upgrading-your-accounts-plan) for free.

**Reply by GhostOf0days** [reply]
Please also check your [profile settings](https://github.com/settings/profile) and make sure you didn't accidentally disable displaying your Github Pro badge.

**Reply by goodfella47** [reply]
Thanks! I successfully upgraded to Github Pro with the $4 off per month coupon.

**Reply by GhostOf0days** [reply]
No problem. Happy to help. 🙂

**Comment by safnimj** [comment]
I think you want to reverify your information 😊
